corehq/ex-submodules/casexml/apps/case/xml/generator.py

```diff
@@ -177,6 +177,8 @@ def add_base_properties(self, element):
     def add_custom_properties(self, element):
         if self.case.external_id:
             element.append(safe_element('external_id', self.case.external_id))
+        if self.case.location_id:
+            element.append(safe_element('location_id', self.case.location_id))
```

Comment:
a test for this change would also be good (maybe in this test file: `casexml.apps.case.tests.test_v2_parsing`)

Reply:
done!